plenum/server/node.py

```diff
@@ -2108,8 +2113,8 @@ def start_catchup(self):
         # Process any already Ordered requests by the replica
 
         if self.mode == Mode.starting:
-            logger.info('{} does not start the catchup procedure '
-                        'because it is already in this state'.format(self))
+            logger.debug('{} does not start the catchup procedure '
```

Comment:
Why don't we have any prefixes for debug-level messages? Are we planning to support them later?